sabnzbd/nzbstuff.py

```diff
@@ -893,8 +893,8 @@ def __init__(self, filename, pp, script, nzb=None,
         if not self.password and self.meta.get('password'):
             self.password = self.meta.get('password', [None])[0]
 
-        # Set nzo save-delay to 6 sec per GB with a max of 5 min
-        self.save_timeout = min(6.0 * float(self.bytes) / GIGI, 300.0)
+        # Set nzo save-delay to 30 seconds
+        self.save_timeout = 30
```

Comment:
This will make it save more often for larger jobs. The larger the job, the more expensive the save is.
I'm OK with increasing the lower limit.